kuwala/core/canvas/src/components/Modals/NodeConfig/NodeConfigModal.js

```diff
@@ -1,29 +1,30 @@
 import React, {useEffect, useState} from "react";
```

Comment:
This modal is specific to data blocks and therefore I think a better name is `Modals/DataBlockConfig/DataBlockConfigModal.js`.